File: rafael/controle-de-fluxo/ADivisivelPorB.cs
class ADivisivelPorB 
{
	static void Main() 
	{
		System.Random gerador = new System.Random();

		int a = (int)(gerador.NextDouble() * 1000);
		int b = (int)(gerador.NextDouble() * 20);

		System.Console.WriteLine("a: " + a);
		System.Console.WriteLine("b: " + b);

		if (a % b == 0) 
		{
			System.Console.WriteLine("É divisível");
		} 
		else 
		{
			System.Console.WriteLine("Não é divisível");
		}
	}
}

File: rafael/controle-de-fluxo/ADivisivelPorB.java
class ADivisivelPorB {
	public static void main(String[] args) {
		int a = (int)(Math.random() * 1000);
		int b = (int)(Math.random() * 20);

		System.out.println("a: " + a);
		System.out.println("b: " + b);

		if (a % b == 0) {
			System.out.println("É divisível");
		} else {
			System.out.println("Não é divisível");
		}
	}
}
